src/acl/acl.test.ts

```diff
@@ -944,11 +1006,9 @@ describe("getFileWithAcl", () => {
           : url === "https://some.pod/"
           ? { Link: '<.acl>; rel="acl"' }
           : { "Content-Type": "text/turtle" };
-      const init: ResponseInit & { url: string } = {
-        headers,
-        url: url as string,
-      };
-      return Promise.resolve(new Response(undefined, init));
+      const response = new Response(undefined, { headers });
+      jest.spyOn(response, "url", "get").mockReturnValue(url as string);
```

Comment:
Why are we spying on the `get` method throughout this series of tests; we don't seem to ever be testing the spyed upon function?

Reply:
This should actually be just a call to `mockResponse`, fixed in daa9984c. You'll note that we don't track the reference to the spied function, here the intent is simply to mock its return value. a95495869002c2eaf54cfd3035d7c93743dc798c should clarify a bit there.